website/docs/components/tag/partials/accessibility/support.md

```diff
@@ -0,0 +1,4 @@
+---
+
+## Support
```

Comment:
```suggestion
## Support

```

Reply:
This is how it is on the [alert page](https://hds-website-ha88s82ai-hashicorp.vercel.app/components/alert?tab=accessibility#support), so I'm inclined to leave this. I think we are likely to replace this before launch anyhow.